astropy/coordinates/tests/test_finite_difference_velocities.py

```diff
@@ -295,24 +295,30 @@ def diff_info_plot(frame, time):
     from matplotlib import pyplot as plt
 
     fig, ((ax1, ax2), (ax3, ax4)) = plt.subplots(2, 2, figsize=(20, 12))
-    ax1.plot_date(
-        time.plot_date, frame.data.differentials["s"].d_xyz.to(u.km / u.s).T, fmt="-"
```

Comment:
I wonder what was the last time anyone called this function. Maybe we should just delete it.

Reply:
`diff_info_plot()`

Reply:
oh yes. Maybe as a follow up PR then, just in case anyone complains about its removal after the fact so we can easily revert it ?

Reply:
`diff_info_plot()` is defined in our tests, so we can be sure normal users are not relying on this. So it really boils down to whether or not @adrn, @mhvk or @StuartLittlefair are interested in keeping it.

Reply:
I've never used it, even though I do sometimes use tests interactively in debugging. I think the chance of knowing of its existence is far too small, so agree with @eerovaher that we should just remove it. But just in case, ping @eteq, who I think introduced it.